tests/_files/RequirementsClassDocBlockTest.php

```diff
@@ -14,6 +14,7 @@
  * @requires OS Linux
  * @requires function testFuncClass
  * @requires extension testExtClass
+ * @requires env TEST_ENV dev|ci
```

Comment:
I would much prefer to see the common regex slashes so it is more obvious what it is. Now this is hidden in the `/%s/i` in `src/Util/Test.php`

Reply:
I agree that it would be better, but OS requirement is also a regex and it hides it the same way. IMHO it's better to keep it consistent and maybe change both of them in some future major release. Let me know what you think.